db/migrate/20221207113235_add_bulk_upload_filename.rb

```diff
@@ -0,0 +1,5 @@
+class AddBulkUploadFilename < ActiveRecord::Migration[7.0]
+  def change
+    add_column :bulk_uploads, :filename, :text
```

Comment:
So i think I'd have to do some reading. My concern would be a browser for whatever reason doesn't send us the filename i think we therefore don't want to hard block here. We just store the name if we can otherwise let it be blank and could render back to the user as `unknown file` or something to that degree

Reply:
Ah yes, that makes sense. We could render the S3 file name, but that's probably not useful either.